src/view.rs

```diff
@@ -94,94 +94,97 @@ use crate::{
     id::Id,
     style::{BoxShadowProp, Style, StyleClassRef},
     view_data::ViewStyleProps,
+    views::stack_from_iter,
     EventPropagation,
 };
 
 pub use crate::view_data::ViewData;
 
-pub type AnyWidget = Box<dyn Widget>;
+pub type AnyView = Box<dyn View>;
 
-pub trait View: Sized {
-    fn view_data(&self) -> &ViewData;
-    fn view_data_mut(&mut self) -> &mut ViewData;
+pub trait IntoAnyView {
+    fn any(self) -> AnyView;
+}
 
-    fn id(&self) -> Id {
```

Comment:
This method is missing in `IntoView`. That matter for types which are `IntoView`, but doesn't implement `View`.